db/migrate/20151103041145_change_sensor_column_name.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeSensorColumnName < ActiveRecord::Migration
+  def change
+    rename_column :sensors, :type, :sensorType
```

Comment:
should modify `db/migrate/20151102162832_create_sensors.rb` instead. Also `type` doesn't work because it's a reserved name used in [polymorphic associations](http://guides.rubyonrails.org/association_basics.html#polymorphic-associations)